yarn.lock

```diff
@@ -7683,12 +7683,12 @@ braces@^2.3.1, braces@^2.3.2:
     split-string "^3.0.2"
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33040&commitId=5796758c59b3328b2f541586a6af4b93b374f42e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|47| Changed|
|Avatar Converged.badgeMask.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|3302| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>